src/Exception.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace JakubOnderka\PhpParallelLint;
+
+class Exception extends \Exception implements \JsonSerializable
```

Comment:
Renaming class with same name (only namespacing) is not very good style and can lead to errors that are difficult to troubleshoot. What if you forgot import? The code may work, but not as intended, and later it's not original author of the commit who has to troubleshoot, and the original author may not even be available.

Also, don't extend exception, but more specific, RuntimeException.

I suggest as name `Exception\ParallelLintException`, and if there are more exceptions in this project, let them extend from that exception. Perhaps even make the root exception class abstract, so more specific exceptions are created.

Reply:
This feedback belongs with PR #44.